stream_alert/apps/_apps/README.rst

```diff
@@ -96,3 +96,66 @@ back on local system
 
   # scp to local host's current directory
   $ scp -i /path/to/<private-key>.pem ec2-user@public.dns.address:~/pip_temp/pip.zip .
+
+Build dependencies on an virtual machine locally (python3.7)
+============================================================
+
+There is a Vagrantfile located in streamalert repo root directory. We can use this Vagrantfile to spin up an Ubuntu 18.04 virtual machine locally to build dependencies for box sdk or aliyun sdk.
+
+Download and install Vagrant if you haven't
++++++++++++++++++++++++++++++++++++++++++++
+
+Please visit `Vagrant download page <https://www.vagrantup.com/downloads.html>`_ for the information (recommend to install the latest Vagrant package).
+
+Bring up the virtual machine
+++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+  # It will take few mins to spin up virtual machine depending on your network speed
+  $ vagrant up
+
+Login virtual machine and build the dependencies
+++++++++++++++++++++++++++++++++++++++++++++++++
+
+.. code-block:: bash
+
+  $ vagrant ssh
+
+  # make sure you create virtual environment with python3.7
+  $ which python3.7
+
+  # Create virtual environment using virtualenvwrapper which is pre-installed
+  $ mkvirtualenv -p /usr/bin/python3.7 test_venv
+
+  # upgrade pip and setuptools if neccessary
+  $ pip install --upgrade pip setuptools
+
+  $ mkdir $HOME/build_temp $HOME/pip_temp
+  $ pip install boxsdk[jwt]==2.6.0 --build $HOME/build_temp/ --target $HOME/pip_temp
+
+  # Replace the `boxsdk[jwt]==2.6.0` below with the desired package & version
+  # For example, the following would update the aliyun dependencies:
+  # pip install aliyun-python-sdk-actiontrail==2.0.0 --build $HOME/build_temp/ --target $HOME/pip_temp
+
+  $ cd $HOME/pip_temp
+  $ find . -name '*.pyc' | xargs rm -rf
+
+  # Install zip package
+  $ sudo apt-get install zip
+  $ zip -r pip.zip .
+  $ exit
+
+Transfer pip.zip file from virtual machine to local host
```

Comment:
same as above comments - too much information for a header here. Simplify it and add a note below with more information
```rst
Transfer Dependencies
+++++++++++++++++++++

Copy the `pip.zip` file from the virtual machine to the local host

.. code-block:: bash

...
```